lib/member_page.rb

```diff
@@ -8,7 +8,10 @@ class MemberPage < Page
   end
 
   field :area do
-    noko.xpath("id('Breadcrumb')/li[5]/a").text.tidy
+    noko.xpath("id('Breadcrumb')/li")[-2]
```

Comment:
If you want to keep this in XPath, you can use noko.xpath("id('Breadcrumb')/li[last() - 1]")